job_executor/model/metadata.py

```diff
@@ -87,7 +87,7 @@ def dict(self, **kwargs) -> dict:
             }
```

Comment:
this dict method might also be removable

Reply:
See comment above.